### ISSUE TITLE
test/performance/test.py prints wrong query on failure

### ISSUE THREAD

**Issue by danielmewes** [rethinkdb/rethinkdb#2897]
This line https://github.com/rethinkdb/rethinkdb/blob/next/test/performance/test.py#L267 should be `print(table_queries[p])`

**Comment by larkost** [comment]
@danielmewes: Is this still an issue? If so could you find an up-to-date line number?

**Comment by danielmewes** [comment]
Yes, https://github.com/rethinkdb/rethinkdb/blob/next/test/performance/test.py#L260
